database/CSVfilewrite/createID.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const writer = csvWriter();
+
+//creates movie csv file
+const idGen = () => {
+  console.time('timing seed');
+  writer.pipe(fs.createWriteStream('ArtilleryId50.csv'));
+  for (let i = 0; i < 50; i++) {
```

Comment:
We may need to adjust this number back up if we figure out the issues regarding dropping data points!